docs/application/native/guides/app-management/sticker.md

```diff
@@ -0,0 +1,427 @@
+# Sticker
+
+
+The Sticker feature provides you with fun experiences to share sticker information. The Sticker applications are composed of a [Sticker provider application](#provider_application) and [Sticker consumer application](#consumer_application). The Sticker provider creates stickers and you can create your own Sticker consumer applications that use the stickers created by the Sticker provider. The Sticker provider application can store sticker details such as URI, keyword, and group name to the Sticker database. The Sticker consumer applications can retrieve sticker information using the Sticker API.
```

Comment:
Minor change, please add article "a" before " [Sticker consumer application](#consumer_application)":

The Sticker feature provides you with fun experiences to share sticker information. The Sticker applications are composed of a [Sticker provider application](#provider_application) and a [Sticker consumer application](#consumer_application).

Reply:
> Minor change, please add article "a" before " [Sticker consumer application](#consumer_application)":  👎 
> 
> The Sticker feature provides you with fun experiences to share sticker information. The Sticker applications are composed of a [Sticker provider application](#provider_application) and a [Sticker consumer application](#consumer_application).

"Consumer applications" is correct instead of "A Sticker consumer application".
Developers can create not one application but also applications.

Reply:
Dear @journey2w 
Please check a fixed patch.